### PR TITLE
[SEC-102] offline flag for parameter values

### PR DESCRIPTION
**Pull request by bhatman**

### CODE REVIEW

File: clrenv/lazy_env.py [hunk 1]
```diff
@@ -3,16 +3,35 @@
...
 from .path import find_environment_path, find_user_environment_paths
 from functools import reduce
 
+logger = logging.getLogger(__name__)
+
+# Flag to prevent clrenv from throwing errors
+#  if it cannot connect to the Parameter Store API.
+OFFLINE_FLAG = 'CLRENV_OFFLINE_DEV'
+
+class LazyMunch(Munch):
```
Comment (bhatman) [inline clrenv/lazy_env.py]: With the offline flag, I'm not sure whether I need to do this lazy-loading anymore. Leaning towards removing.

**Review by youssefm — APPROVED** [review]
:shipit: 👍

**Comment by bhatman** [comment]
@youssefm I just added lazy-loading in again, because I was seeing an error when I tried to `setenv` otherwise (since the AWS credentials hadn't been loaded yet). The graph of events is:

```
setenv
    -> loads `clrenv` to check the environment mode
        -> without lazy-loading, requires AWS credentials to fetch params from SSM
    -> loads AWS credentials depending on environment mode
```

**Comment by bhatman** [comment]
Per discussion, I have removed the lazy-loading ability. Instead, we will load AWS credentials before clrenv, and rely on the offline flag when needed for local dev.

**Pull request merged**